test/cea/cea608_memory_unit.js

```diff
@@ -5,238 +5,301 @@
  */
 
 describe('Cea608Memory', () => {
+  const ceaUtils = shaka.test.CeaUtils;
```

Comment:
Style nit: Since this is a class reference, the alias should start with a capital, just like the original.